src/content/docs/en/pages/services/integration-services/integration-services.mdx

```diff
@@ -6,35 +6,38 @@ namespace: documentation_services_integration_services
 permalink: /documentation/services/solutions-lab/integration-services/
 ---
 
-**Integration Services** is the fastest and safest way to migrate your content, web applications, and APIs to Azion, no matter how complex and resource-intensive your infrastructure is. The Integration Services team will activate the Azion products and services you've contracted and provide the required tools, documentation, and training for your success using the Azion Platform.
+**Integration Services** is the fastest and safest way to migrate your content, web applications, and APIs to Azion, no matter how complex and resource-intensive your infrastructure is. Our team will provide the required documentation for your success using the Azion Platform.
 
-| Plan | Hours/Year | Additional Hours Available |
-| ---- | ---------- | -------------------------- |
-| Developer | Not available  | Not available |
-| Business | 5 hours | Yes |
-| Enterprise | 20 hours | Yes |
-| Mission Critical | 60 hours | Yes |
+| Service Plan      | Hours          | Additional hours available |
+| ----------------- | -------------- | -------------------------- |
+| Business          | 5 hours/year   | Yes                        |
+| Enterprise        | 20 hours/year  | Yes                        |
+| Mission Critical  | 60 hours/year  | Yes                        |
 
-For more detailed information, see [Service Plans options](/en/documentation/services/service-plans/).
+For more detailed information, see [Azion Service Plans](https://www.azion.com/en/pricing/#plans).
 
 An Azion expert will be available to support you throughout the process via:
 
 1. Activation of the Azion services contracted as described in the *Service Order*.
 2. Gathering of technical and business requirements with the customer.
 3. Real-Time Manager (RTM) usage training.
 4. Configuration of Azion services for the customer.
-5. Review and optimization of service configurations according to customer demands.
+5. Review and optimization of service configurations according to customer demands during the launch.
 
 The Azion specialist will follow an integration process consisting of the steps below:
 
+import TableContainer from "~/components/Table/TableContainer.astro"
+
+<TableContainer>
```

Comment:
Genuine question. If tables and simple <p> already work, what is the point of `<TableContainer>`?

Reply:
It makes it so the first column is not as squished, like in the old variables table in Rules Engine.